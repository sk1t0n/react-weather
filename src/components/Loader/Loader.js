import React from 'react';
import './Loader.css';

function Loader(props) {
  return <div className="lds-dual-ring"></div>;
}

export { Loader };
