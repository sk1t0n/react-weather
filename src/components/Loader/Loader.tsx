import React from 'react';
import './styles.css';

const Loader: React.FC = () => (
  <div className="lds-dual-ring"></div>
);

export { Loader };
