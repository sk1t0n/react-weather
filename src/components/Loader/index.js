import React from 'react';
import './styles.css';

function Loader(props) {
  return <div className="lds-dual-ring"></div>;
}

export { Loader };
