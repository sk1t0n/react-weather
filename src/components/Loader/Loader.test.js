import React from 'react';
import renderer from 'react-test-renderer';
import { Loader } from './Loader';

test('Render correctly loader component', () => {
  const component = renderer.create(<Loader />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
