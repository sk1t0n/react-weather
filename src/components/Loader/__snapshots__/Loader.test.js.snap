// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly loader component 1`] = `
<div
  className="lds-dual-ring"
/>
`;
